### ISSUE TITLE
Check Travis

### ISSUE THREAD

**Issue by ssardina** [ssardina-agts/agtcity-server#15]
The readme includes:

```
[![Build Status](https://travis-ci.org/agentcontest/massim_2018.svg?branch=master)](https://travis-ci.org/agentcontest/massim_2018)
```

but this is for the original server right?

**Comment by ayoung012** [comment]
yes, we can remove this

**Comment by ssardina** [comment]
but can we replace it with one of us? or is too difficult? or not worth it?

**Comment by ayoung012** [comment]
Hmm yes good idea, there is currently no way to get the full server jar
without compiling from source.

Could we create a GitHub release instead?

**Comment by ssardina** [comment]
Totally, we could and should, as they do in the original server: 

https://github.com/agentcontest/massim_2018/releases

However, isn't Travis a service to compile and verify there are no errors? It is not to host releases right?

**Comment by ayoung012** [comment]
It does not host the release itself, but it can compile, verify and
automatically upload a Release to GitHub on successful tagged builds.

https://docs.travis-ci.com/user/deployment/releases/

Whose account shall we use? I know Dhirendra has one.

**Comment by ssardina** [comment]
It is free right? (at least if used not a lot)... Let me check...

**Comment by ssardina** [comment]
![Uploading image.png…]()

**Comment by ayoung012** [comment]
I don't have the repo rights to enable it here. You will have to do it

**Comment by ssardina** [comment]
really? I am on it, but let me give you more permissions then.

**Comment by ssardina** [comment]
![image](https://user-images.githubusercontent.com/2368384/108691112-3ced8f80-754f-11eb-91f8-1e539b2332f7.png)

**Comment by ssardina** [comment]
done you are admin now

**Comment by ssardina** [comment]
The [travis.yml](https://github.com/ssardina-agts/agtcity-server/edit/master/.travis.yml) needs the API key to upload the release to GitHub.

is that the key from GitHub or from Travis? mmm

**Comment by ssardina** [comment]
OK it is the GitHub one, makes sense, but seems it is not good to write it plain. Travis has some type of interface to produce these files:

![image](https://user-images.githubusercontent.com/2368384/108696524-c607c500-7555-11eb-8268-1fc1d7b45808.png)


gee, complex eh..

**Comment by ayoung012** [comment]
Hmm, yes, and seems you can only associate these tokens with a user account, and not an organisation. For now I will just create a release manually

**Comment by ayoung012** [comment]
Are we ready for a release? Do we want to mess with any more config?

**Comment by ssardina** [comment]
Agree. let's dot it manually. I have been looking at Travis and I really cannot figure it out well how it should be configured. And they are hanging to a new way of authentication.

It is recommended to not touch the yml file manually but use the command line tool:

https://github.com/travis-ci/travis.rb#command-line-client


Let's do the release manually. But we need a release it can be downloaded and run "off-the-shelf". I don't think just the JAR file is enough, at least we need the configuration files for the server
